src/status_im/ui/components/svgimage.cljs

```diff
@@ -0,0 +1,49 @@
+(ns status-im.ui.components.svgimage
+  (:require [status-im.ui.components.react :as react]
+            [reagent.core :as reagent]
+            [status-im.utils.platform :as platform]))
+
+(defn html [uri width height]
+  (str
+   "<!DOCTYPE html>\n
+   <html>
+   <head>
+   <style type=\"text/css\">
+   img {
+        display: block;
+        max-width:" width "px;
+        max-height:" height "px;
+        width: auto;
+        height: auto;
+        margin: auto;
+    }
+    div {
+      width:" width "px;
+      height:" height "px;
+      display:table-cell;
+      vertical-align:middle;
+      text-align:center;
+    }
+    body {margin: 0;}
+
+    </style>
+    </head>
+    <body>
+    <div>
+    <img src=" uri " align=\"middle\" />
```

Comment:
is URI sanitized there?

Reply:
potential risks that I see:
- code execution injecting some unwanted javascript here
- deep-link link to the app internal resources and sending them to a potentially malicious source
- user tracking
- CSRF attacks

Reply:
ping me if these all aren't possible in that particular case or we handle them somehow

Reply:
@mandrigin these dapps are whitelisted, and anyway we load this in the webview sandbox, so it may be any url

Reply:
so even if the url would be something like `"></img><script>alert("hello here");</script>`, it is still fine?
and there is no way to make requests pretending these are requests fro the real user?

Reply:
yeah you're right, lets check the url ,i'll make a fix

Reply:
@mandrigin done